recipe/meta.yaml

```diff
@@ -36,8 +37,8 @@ outputs:
         - mypy_extensions >=1.0.0
         - pathspec >=0.9.0
         - tomli >=1.1.0  # [py<311]
-        - types-psutil
-        - types-setuptools
+        - types-psutil 5.9.0
+        - types-setuptools 69.0.0.20240125
```

Comment:
Keep these Python dependencies without versions in `host`, see https://github.com/python/mypy/blob/v1.17.1/pyproject.toml#L13-L14